deploy/quickstart.sh

```diff
@@ -7,17 +7,26 @@ set -e
 #  Upstream containers: ./quickstart.sh
 #  Downstream containers: ./quickstart.sh --downstream-secret=~/6340056-cloudops-pull-secret.yaml
 oc login -u system:admin
-oc new-project sa-telemetry
+#if [ -z "$(oc projects | grep -o 'sa-telemetry')" ]; then #project does not exist
```

Comment:
I don't love comments in files that aren't intending to be added, so hopefully this will go away before merge.

Reply:
Left it in there on accident, I will make another sweep open merging